src/postgres/postgresConnectionStrings.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ConnectionOptions, parse } from "pg-connection-string";
+import { ParsedConnectionString } from "../ParsedConnectionString";
+import { nonNullProp } from "../utils/nonNull";
+
+export async function parsePostgresConnectionString(connectionString: string): Promise<ParsedPostgresConnectionString> {
+    const config: ConnectionOptions = parse(connectionString.trim());
+    return new ParsedPostgresConnectionString(connectionString, config);
+}
+
+export async function createPostgresConnectionString(host: string, username?: string | undefined, password?: string | undefined): Promise<ParsedPostgresConnectionString> {
+    const port = '5432';
```

Comment:
You hard-code 5432 in several places. Make a `const` for it. If you use it in several files, stick it in the `constants.ts` file